pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -152,9 +153,30 @@ let
     inherit (upstream-info) version;
     inherit packageName buildType buildPath;
 
-    src = fetchurl {
+    src = fetchzip {
+      name = "chromium-${version}.tar.zstd";
       url = "https://commondatastorage.googleapis.com/chromium-browser-official/chromium-${version}.tar.xz";
       inherit (upstream-info) sha256;
+
+      nativeBuildInputs = [ zstd ];
+
+      postFetch = ''
+        echo removing unused code from tarball to stay under hydra limit
+        rm -r $out/third_party/{rust-src,llvm}
+
+        echo moving remains out of \$out
+        mv $out source
+
+        echo recompressing final contents into new tarball
+        # try to make a deterministic tarball
+        tar \
+          --use-compress-program "zstd -T0" \
```

Comment:
Please don't use blind auto-detection of cores.  Here I'd suggest simply `-T$NIX_BUILD_CORES`

You know, the issue is when you have many-core machine.  For example, on Hydra we have 80-core machines with --max-jobs 40, so 40*80 is quite a lot.

Reply:
Whoops^

I know of the variable and it's purpose.
I just genuinely forgot about adding that yesterday.

Thanks for catching that!

(applied `-T$NIX_BUILD_CORES`)